sdk/translation/ai-translation-text-rest/src/customClient.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I couldn't find the generated createClient function. Was it removed manually after generation? We shouldn't really modify anything under `generated/` otherwise those changes will be lost the next time codegen is executed.

Reply:
I was using tsp compile with TS emitter. We can revisit this for GA.

Reply:
I am not sure whether any manual changes have been done. We will see with next iteration before GA.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/25546